internal/batches/executor.go

```diff
@@ -358,6 +358,19 @@ func (x *executor) do(ctx context.Context, task *Task) (err error) {
 		log.Close()
 	}()
 
+	if x.features.selectSearchOperator {
```

Comment:
Doesn't this also need the check for whether the `steps` actually use templating?